enterprise/frontend/test/metabase-enterprise/formatting/whitelabel.cy.spec.js

```diff
@@ -192,12 +206,9 @@ describeWithToken("formatting > whitelabel", () => {
     });
   });
 
-  describe("Changes to logo work", () => {
-    it("should add a logo", () => {
-      signInAsAdmin();
-      cy.visit("/admin/settings/whitelabel");
-
-      cy.server();
+  describe("company logo", () => {
+    beforeEach(() => {
+      cy.log("**Add a logo**");
       cy.readFile(
         "enterprise/frontend/test/metabase-enterprise/_support_/logo.jpeg",
         "base64",
```

Comment:
Could the PUT request below get cleaned up, so it's only `value`?

Reply:
I think it can. This endpoint isn't covered in documentation, but I'll take a look at the source code and try to clean it up.

Reply:
It's this `https://github.com/metabase/metabase/blob/master/docs/api-documentation.md#put-apisettingkey`
Basically you just need to supply value, but I'm sure some of the other stuff helps with fallback/etc.

Reply:
Oh, thank you! I was searching for the literal endpoint. This makes sense :)